src/components/AppHeaderMenu.vue

```diff
@@ -27,6 +27,9 @@ export default {
 
   methods: {
     clickHandler(idx) {
+      //collapse navbar before changing active item. Click on the button to do so
+      document.querySelector(".navbar-toggler").click();
```

Comment:
Best not to do it like this. This activates the toggle for laptop size too (I presume the element is just hidden by a display none and the js is nevertheless available)
It would be better to use the controls bootstrap gives us since the toggle action is anyway something done by them

Reply:
See if there's a way to check the size and do it only for mobile sizes. (currently it sort of blinks when I press on the laptop)